src/test/errors-analysis.test.ts

```diff
@@ -119,6 +119,37 @@ test(
   })
 );
 
+test.only(
```

Comment:
Remove only

Reply:
Eep! Done

Also added an eslint pass to prevent this in the future and verified that it flagged this instance.

Reply:
Nice